sections/article.php

```diff
@@ -9,5 +9,13 @@
             </div>
         </div>
     </a>
-</article>
-
+    <?php
+    if( $_SERVER['REQUEST_URI'] == '/pages/admin_panel_page.php' ) {
```

Comment:
I guess you can change this condition and check if current user is admin, rather than checking for page URI